src/EditorFeatures/TestUtilities/PullMemberUp/PullMemberUpCodeActionTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp
+{
+    public abstract class PullMemberUpCodeActionTest : AbstractCodeActionTest
+    {
+        internal Task TestWithPullMemberDialogAsync(
+            string initialMarkUp,
+            string expectedResult,
+            IEnumerable<(string, bool)> selection = null,
```

Comment:
name these tuple prameters.  i have no idea what they mean.  (presumably the name of hte member, and if it was checked or not?)  But if it's that, why not just have the names of hte members that are checked.

Reply:
> name these tuple prameters.

please still do this.